cmd/relayer/main.go

```diff
@@ -64,6 +72,27 @@ var (
 			Value: "info",
 			Usage: "Set log level: one of [error|warn|info|debug]",
 		},
+		&cli.BoolFlag{
+			Name:  flagWithNetwork,
+			Value: false,
+			Usage: "Run the relayer with p2p network capabilities",
+		},
+		&cli.StringFlag{
+			Name:  flagLibp2pKey,
+			Usage: "libp2p private key",
+			Value: fmt.Sprintf("%s/node.key", os.TempDir()),
+		},
+		&cli.UintFlag{
+			Name:  flagLibp2pPort,
+			Usage: "libp2p port to listen on",
+			Value: defaultLibp2pPort,
+		},
+		&cli.StringSliceFlag{
+			Name:    flagBootnodes,
+			Aliases: []string{"bn"},
+			Usage:   "libp2p bootnode, comma separated if passing multiple to a single flag",
+			EnvVars: []string{"SWAPD_BOOTNODES"},
```

Comment:
The environment variable above is `SWAPD_BOOTNODES` which probably isn't what you intended.